addons/mail/static/src/core/common/chat_window.js

```diff
@@ -95,6 +95,9 @@ export class ChatWindow extends Component {
             return;
         }
         ev.stopPropagation(); // not letting home menu steal my CTRL-C
+        if (ev.target.closest(".o-mail-AutoresizeInput")) {
+            return;
```

Comment:
I suggest to `markEventHandled` in the rename flow, and check `isEventHandled` there.